compiler/natives/src/net/http/cookiejar/example_test.go

```diff
@@ -0,0 +1,7 @@
+// +build js
+
+package cookiejar
```

Comment:
I think this needs to be `package cookiejar_test`. The example is in an external test package.

See https://github.com/golang/go/blob/go1.8/src/net/http/cookiejar/example_test.go#L5.